dbt/models/default/schema/default.vw_pin_sale.yml

```diff
@@ -80,7 +80,7 @@ models:
             - year
           allowed_duplicates: 2
           config:
-            error_if: ">7319"
+            error_if: ">4500"
```

Comment:
**question:** Is this supposed to be `7500`? The "bump" verb in the title makes it seem so. Or are we dropping the threshold due to changes in the underlying view?